enarx-keep-sev/src/x86_64.rs

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use x86_64::structures::paging::page_table::PageTable;
+
+pub const PML4_START: u64 = 0x9000;
+pub const PDPTE_START: u64 = 0xA000;
```

Comment:
What are these magic numbers? Where do they come from?

Reply:
@npmccallum `PML4_START` is the guest physical address of the root of the page table structure. `PDPTE_START` is the first top-level page table entry. However, as I was typing these definitions into my next revision, your question raised a question of my own:

@haraldh should we move the page table setup into the shim? In `demo2020` the keep sets up the page tables and then loads the address into the guest's `cr3` register when we're provisioning a vCPU. I think anything that can be done in the trusted microkernel should be done in the trusted microkernel.

Reply:
If we want 64bit mode, we need a page table for the CPU to be allowed to read instructions from `rip`. The shim can't di it, if it can't run. The addresses of the pages could be more or less dynamic. You just have to fill in the entry with the correct address.